tilequeue/process.py

```diff
@@ -512,6 +522,34 @@ def _calculate_scale(scale, coord, nominal_zoom):
         return scale
 
 
+def _load_inline_layer(layer_path):
+    # we can optionally load geojson layers from file all we need is a file path. we expect the coordinates to already
+    # be in mercator projection and we only fill out the minimal structure of the feature tuple, ie there's no ID. if
+    # the path is given and the file exists, we expect to be able to load the layer's features. so if the json is
+    # malformed or shapely can't make sense of the geom then we'll raise
+    features = []
+
+    # has to exist and has to be geojson for now
+    if not os.path.isfile(layer_path) or not layer_path.endswith('.geojson'):
+        return features
+
+    # load the geojson into a shapely geometry collection
+    with open(layer_path) as fh:
+        fc = json.load(fh)
+        # skip if this isnt pseudo mercator
+        if fc['crs']['properties']['name'] != 'urn:ogc:def:crs:EPSG::3857':
```

Comment:
Same concern, is there any log we can emit to tell whether it happens, otherwise it is hard to debug if it not working as expected?

Reply:
here we can log as it was clear the person meant to load it but screwed something up. maybe same for the file extension